csharp/Squidex.ClientLibrary/Squidex.ClientLibrary/EnrichedEvents/EnrichedAssetEvent.cs

```diff
@@ -0,0 +1,77 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschraenkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System;
+using Squidex.ClientLibrary.Management;
+
+namespace Squidex.ClientLibrary.EnrichedEvents
+{
+  public sealed class EnrichedAssetEvent : EnrichedUserEventBase, IEnrichedEntityEvent
+  {
+    public EnrichedAssetEventType Type { get; set; }
+
+    public string Id { get; set; }
+
+    public DateTimeOffset Created { get; set; }
+
+    public DateTimeOffset LastModified { get; set; }
+
+    public string CreatedBy { get; set; }
+
+    public string LastModifiedBy { get; set; }
+
+    public string MimeType { get; set; }
+
+    public string FileName { get; set; }
+
+    public long FileVersion { get; set; }
+
+    public long FileSize { get; set; }
+
+    public int? PixelWidth { get; set; }
+
+    public int? PixelHeight { get; set; }
+
+    public AssetType AssetType { get; set; }
+
+    public bool IsImage { get; set; }
+
+    public override long Partition { get; set; }
+
+    public string CreatedById
+    {
+      get
+      {
+        return CreatedBy == null ? null : CreatedBy.Contains(":") ? CreatedBy.Split(':')[1] : null;
```

Comment:
Can we create a helper method for this property?

I also think we should mark these properties As JsonIgnore, in case somebody forwards them.